Extensions/ClaimPrincipalExtension.cs

```diff
@@ -12,16 +12,16 @@ public static class ClaimsPrincipalExtension
         /// <returns>A string corresponding to an account identifier as defined in <see cref="Microsoft.Identity.Client.AccountId.Identifier"/></returns>
         public static string GetMsalAccountId(this ClaimsPrincipal claimsPrincipal)
         {
-            string userObjectId = GetObjectId(claimsPrincipal);
-            string tenantId = GetTenantId(claimsPrincipal);
+            var userObjectId = GetObjectId(claimsPrincipal);
```

Comment:
I even less see the point for strings :)

Reply:
I understand, can be replaced with explicit type. From code perspective, it is the same, from reading and code style perspective, it defers. :)